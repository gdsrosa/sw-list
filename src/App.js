import React from 'react';

import Characters from './components/Characters';
import Starships from './components/Starships';
import './styles/index.scss';

const App = () => (
  <div>
    <Characters />
    <Starships />
  </div>
);

export default App;
