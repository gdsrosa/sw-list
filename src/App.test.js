import React from 'react';
import { shallow } from 'enzyme';

import App from './App';
// eslint-disable-next-line no-unused-vars
import setupTest from './setupTest';

describe('App Component', () => {
  it('should render self and subcomponents', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
