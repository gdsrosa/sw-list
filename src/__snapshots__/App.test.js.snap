// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render self and subcomponents 1`] = `ShallowWrapper {}`;
